cmd/k.go

```diff
@@ -15,7 +15,8 @@ var kcmd = &cobra.Command{
 	Long:  "Work with Kubernetes experiments",
 }
 
-// addExperimentGroupFlag adds the experiment group flag
+// TODO: is group still needed?
+// addExperimentGroupFlag adds the test group flag
 func addExperimentGroupFlag(cmd *cobra.Command, groupP *string) {
 	cmd.Flags().StringVarP(groupP, "group", "g", driver.DefaultExperimentGroup, "name of the experiment group")
 }
```

Comment:
Group is now the name of the experiment right? Does this need to be renamed?

Reply:
Agree that this flag is no longer needed.

Reply:
Actually, how is the `--group` option used by `iter8 k run`? Do we still need it for that or can we get it from a name instead?  If not, it might still be needed.